cytosnake/cli/cmd.py

```diff
@@ -96,9 +98,27 @@ def run_cmd() -> None:
                 print(run_doc)
                 sys.exit(0)
 
+            # check if benchmarking is enabled. if so, create benchmarks folder
+            gen_configs = load_cytosnake_configs()
+            if gen_configs["enable_profiling"]:
+                # create benchmark directory, raise error if
+                logging.info("Memory profiling enabled")
+                benchmark_folder = pathlib.Path("benchmarks")
+                benchmark_folder.mkdir(exist_ok=True)
+
+                # update _paths.yaml with path to benchmark folder
+                paths_configs_path = (
+                    cyto_paths.get_meta_path() / "_paths.yaml"
+                ).resolve(strict=True)
+                update_config(
+                    config_file_path=paths_configs_path,
+                    new_key="benchmarks",
```

Comment:
Hey @d33bs , are you referring to a placeholder inside the `_paths.yaml` file?

Reply:
@axiomcura, thanks, yes in the YAML file is what I was thinking about with that comment. The challenge with dynamically added properties is that they sometimes can become a surprise when working on updates to the content. I trust your best judgement on whether this is appropriate, as sometimes the YAML content is never or infrequently looked at by a human and is instead only for process control.

Reply:
Ah! good point! However, since there is only one configuration parameters that is dynamically being updated at runtime, I'll leave it as it is for now. If the number of dynamically changing configurations increases, then I will implement that approach! Thanks!